src/Microsoft.AspNetCore.SignalR/DefaultHubLifetimeManager.cs

```diff
@@ -140,6 +140,14 @@ private string GetInvocationId()
             return invocationId.ToString();
         }
 
+        public override Task InvokeAllExceptAsync(List<string> excludedIds, string methodName, object[] args)
+        {
+            return InvokeAllWhere(methodName, args, connection =>
+            {
+                return !excludedIds.Contains(connection.ConnectionId);
```

Comment:
Yeah, using a `HashSet<string>` for `excludedIds` would be faster, since it has a O(1) contains test